rero_mef/agents/mef/serializers.py

```diff
@@ -64,21 +65,12 @@ def serialize(self, pid, record, links_factory=None, **kwargs):
         :param links_factory: Factory function for record links.
         """
         rec = record
-        if request and request.args.get('resolve'):
-            rec = record.replace_refs()
-            # because the replace_refs loose the record original model. We need
-            # to reset it to have correct 'created'/'updated' output data
+        if request:
+            rec = rec.add_information(
+                resolve=request.args.get('resolve'),
```

Comment:
You should return a default `False` value to be perfectly correct.
Also, if you send `resolve=0` in query string, this should be evaluated as `True` by python because 'request.args.get()' return a string and `bool('0')` return `True`.... (sometimes python is pitty)